interfaces/tracing/v0/README.md

```diff
@@ -0,0 +1,66 @@
+# `tracing`
+
+## Usage
+
+This relation interface describes the expected behavior of any charm claiming to be able to provide or consume a tempo-compliant tracing server.
+
+## Direction
+
+Tracing is done in a push-based fashion. The receiving endpoint of the tracing backend, also referred to as an ingester, can support a number of different protocols, such as [otlp-grpc](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlpgrpc) and [otlp-http](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlphttp).
+The tracing backend **provides**, for each protocol it supports, an endpoint at which the server is ready to accept that protocol. So the directionality of the relation flows from the observer: the trace ingester (a Tempo(-compliant) backend), to the observed: the application producing the traces, such as a mattermost charm.
+
+
+This follows the convention of other push models such as `loki_push_api`.
+
+We call the data structure that is exchanged via this interface 'tracing backend'.
+
+```mermaid
+flowchart LR
+    Provider -- TracingBackend --> Requirer
+```
+
+## Behavior
+### Provider
+
+- Is expected to publish the hostname at which the server is reachable.
```

Comment:
"Hostname" seems to restricting as it excludes scheme, path, ports etc.
"Address" is too ambiguous.
How about URL?

```suggestion
- Is expected to publish the URL at which the server (ingester) is reachable.
```